script/vagrant-provision

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+sudo apt-get update
+
+cd /vagrant/bluetail
+
+#fix dpkg-preconfigure error
+export DEBIAN_FRONTEND=noninteractive
+
+# Install the packages from conf/packages
+xargs sudo apt-get install -qq -y < conf/packages
+xargs sudo apt-get install -qq -y < conf/dev_packages
+# Install some of the other things we need that are just for dev
+sudo apt-get install -qq -y ruby-dev libsqlite3-dev build-essential
+
+# Create postgres database and user.
+sudo -u postgres psql -c "CREATE DATABASE bluetail"
```

Comment:
@SimKennedy What do you think about these two lines? _Really_ they should be using the database name, user, and password from the `DATABASE_URL` environment variable. But that’d mean either passing those things as individual environment variables, or trying to parse them (eek!) out of the `DATABASE_URL` variable.

I can’t see us changing the defaults away from `bluetail:bluetail@bluetail` but I wonder whether this will catch us out one day?

Reply:
Hah! yeah, I don't fancy parsing in bash.. but it is probably best. I think we're pretty safe though.. The provisioning is meant to be for the defaults and vagrant shouldn't stray from that really. If someone wants to use a custom database config they won't be using vagrant to provision it for them.